pkg/koordlet/resmanager/metrics_query.go

```diff
@@ -30,22 +31,91 @@ var (
 	timeNow = time.Now
 )
 
-func (r *resmanager) collectNodeMetricsAvg(windowSeconds int64) metriccache.NodeResourceQueryResult {
-	queryParam := generateQueryParamsAvg(windowSeconds)
-	return r.collectNodeMetric(queryParam)
+var (
+	ErrorMetricNotFound = errors.New("metric not found")
+)
+
+func (r *resmanager) collectorNodeCPUMetricLast() (float64, error) {
```

Comment:
collectorNodeXXX -> collectNodeXXX

it seems that we can merge all this collectNodeXXX functions into two, e.g.

collectNodeMetric(start, end, queryMeta[cpu or memory])
collectNodeMetricLast(queryMeta[cpu or memory])